external/include/protocols/ftpc.h

```diff
@@ -88,7 +88,7 @@
 #endif
 
 #ifndef CONFIG_FTP_TMPDIR
-#define CONFIG_FTP_TMPDIR "/tmp"
+#define CONFIG_FTP_TMPDIR "/mnt"
```

Comment:
please check the file permission. it's changed.

Reply:
Thanks. I Fixed.